[2023-12-12T18:00:01.467+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ppp_via_foia-v01.insert_fk_into_fact_table.insert_fk_col_in_fact_table_Veteran scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2023-12-12T18:00:01.499+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ppp_via_foia-v01.insert_fk_into_fact_table.insert_fk_col_in_fact_table_Veteran scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2023-12-12T18:00:01.507+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-12-12T18:00:01.538+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_fk_col_in_fact_table_Veteran> on 2023-12-01 00:00:00+00:00
[2023-12-12T18:00:01.545+0000] {standard_task_runner.py:57} INFO - Started process 1998 to run task
[2023-12-12T18:00:01.556+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ppp_via_foia-v01.insert_fk_into_fact_table', 'insert_fk_col_in_fact_table_Veteran', 'scheduled__2023-12-01T00:00:00+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/ppp_via_foia.py', '--cfg-path', '/tmp/tmpxbnqs806']
[2023-12-12T18:00:01.562+0000] {standard_task_runner.py:85} INFO - Job 83: Subtask insert_fk_col_in_fact_table_Veteran
[2023-12-12T18:00:01.704+0000] {task_command.py:416} INFO - Running <TaskInstance: ppp_via_foia-v01.insert_fk_into_fact_table.insert_fk_col_in_fact_table_Veteran scheduled__2023-12-01T00:00:00+00:00 [running]> on host edee10095410
[2023-12-12T18:00:01.848+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='dunnbc22' AIRFLOW_CTX_DAG_ID='ppp_via_foia-v01.insert_fk_into_fact_table' AIRFLOW_CTX_TASK_ID='insert_fk_col_in_fact_table_Veteran' AIRFLOW_CTX_EXECUTION_DATE='2023-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-01T00:00:00+00:00'
[2023-12-12T18:00:01.853+0000] {sql.py:274} INFO - Executing: -- params: table_name, dimension_col

/*
Steps 1-3 were completed in the create_dim_table.sql 
file & associated PostgresOperator
*/


-- Step 4: Add foreign key column to fact table
ALTER TABLE ppp_via_foia_pipe
ADD COLUMN Veteran_id INTEGER;

-- Step 4: Update foreign key column to fact table with 
-- appropriate id (that matches with dim table)
UPDATE ppp_via_foia_pipe
SET Veteran_id = dimension_table_Veteran.id
FROM dimension_table_Veteran
WHERE COALESCE(ppp_via_foia_pipe.Veteran::text, '') = COALESCE(dimension_table_Veteran.Veteran::text, '');
[2023-12-12T18:00:01.869+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-12T18:00:01.881+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-12T18:00:01.888+0000] {sql.py:418} INFO - Running statement: -- params: table_name, dimension_col

/*
Steps 1-3 were completed in the create_dim_table.sql 
file & associated PostgresOperator
*/


-- Step 4: Add foreign key column to fact table
ALTER TABLE ppp_via_foia_pipe
ADD COLUMN Veteran_id INTEGER;

-- Step 4: Update foreign key column to fact table with 
-- appropriate id (that matches with dim table)
UPDATE ppp_via_foia_pipe
SET Veteran_id = dimension_table_Veteran.id
FROM dimension_table_Veteran
WHERE COALESCE(ppp_via_foia_pipe.Veteran::text, '') = COALESCE(dimension_table_Veteran.Veteran::text, '');, parameters: None
[2023-12-12T18:02:20.219+0000] {sql.py:427} INFO - Rows affected: 808164
[2023-12-12T18:02:20.244+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ppp_via_foia-v01.insert_fk_into_fact_table, task_id=insert_fk_col_in_fact_table_Veteran, execution_date=20231201T000000, start_date=20231212T180001, end_date=20231212T180220
[2023-12-12T18:02:20.282+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-12T18:02:20.306+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
